k4FWCore/scripts/k4run

```diff
@@ -27,8 +27,7 @@ def load_file(file):
 
 
 def add_arguments(parser, app_mgr):
-  configurables = app_mgr.allConfigurables.values()
-  for conf in configurables:
+  for conf in dict.fromkeys(app_mgr.allConfigurables.values()):
```

Comment:
I think this (or alternatively `list(...)`) is a bit cleaner and achieves the same when I tested with your file
```suggestion
  # length increases when properties of an algorithm with tools are inspected
  # see https://github.com/key4hep/k4FWCore/pull/138
  for conf in tuple(app_mgr.allConfigurables.values()):
```